Assets/Scripts/Managers/HighlightManager.cs

```diff
@@ -131,6 +148,29 @@ private void ResetSelection(GameState gameState)
                 }
 
             }
+            // temp implementation
+            if (selectedPlayer == null)
+            {
+                return;
+            } else
+            {
+                selectedPlayer.UnRenderHand();
+            }
```

Comment:
If its a temp implementation, please remove it or find a cleaner solution for this

Reply:
(This line can be simplified to one line btw)
` selectedPlayer?.UnRenderHand();`